Sources/Testing/Running/Configuration+EventHandling.swift

```diff
@@ -20,6 +20,9 @@ extension Configuration {
   /// `eventHandler` but this method may also be used as a customization point
   /// to change how the event is passed to the event handler.
   func handleEvent(_ event: borrowing Event, in context: borrowing Event.Context) {
-    eventHandler(event, context)
+    var contextCopy = copy context
```

Comment:
How heavyweight do we think this copy operation could be? For example, in scenarios where there's a selected or skipped test filter applied with many entries?

Reply:
Not very. Copy-on-write means that we incur minimal actual memory traffic here.

Reply:
For those at home reading this PR or these comments: the reason `Event.Context` is `borrowing` is to emphasize that its contents—instances of `Test`, `Test.Case`, and now `Configuration`, aren't _free_ and that copies should be avoided where possible. That doesn't preclude the possibility of _any_ copies, just forces us to think about them when they are needed.